test/e2e/magellan-gutenberg-coblocks.json

```diff
@@ -0,0 +1,13 @@
+{
+	"mocha_tests": [ "specs-gutenberg/wp-calypso-gutenberg-coblocks-spec.js" ],
```

Comment:
I _think_ we should still run all GB tests here as CoBlocks could in theory affect Gutenberg in unexpected ways.

Reply:
> What if, instead of creating a separate build, we add another toggle to the Gutenberg e2e build to set COBLOCKS_EDGE, like we set GUTENBERG_EDGE? Would that work?

That exactly what we need! 🙌

Reply:
Done